coderd/coderd.go

```diff
@@ -440,6 +440,13 @@ func New(options *Options) *API {
 					CurrentVersion:               buildinfo.Version(),
 					WorkspaceProxiesFetchUpdater: *(options.WorkspaceProxiesFetchUpdater).Load(),
 				},
+				ProvisionerDaemons: healthcheck.ProvisionerDaemonsReportOptions{
+					CurrentVersion:       buildinfo.Version(),
+					CurrentAPIVersion:    provisionersdk.VersionCurrent,
+					ProvisionerDaemonsFn: options.Database.GetProvisionerDaemons,
+					TimeNowFn:            dbtime.Now,
```

Comment:
This could default to `dbtime.Now` so it's optional. I guess the same logic could be applied to most fields though.